### ISSUE TITLE
Is there a secondary way to run ez-text2video without conda?

### ISSUE THREAD

**Issue by SuperGamer001** [kpthedev/ez-text2video#13]
I want to run this tool, but I don't have conda (nor really want to download it)
Is there another way to use this without conda?

**Comment by kpthedev** [comment]
Definitely, you could try a venv and then manually pip install all the packages that are in the `environment.yaml` file.
